config.rb

```diff
@@ -38,6 +38,8 @@
 activate :relative_assets
 set :relative_links, true
 
+config[:api_endpoint] = "https://api.tito.io/v3/"
```

Comment:
Is this the right endpoint? It doesn't matter too much right now as I can switch it to whatever if there is a single source of truth in the docs.